pkgs/games/fteqw/generic.nix

```
@@ -14,13 +14,13 @@
 
 stdenv.mkDerivation {
   inherit pname buildFlags buildInputs nativeBuildInputs postFixup;
-  version = "unstable-2023-08-03";
+  version = "unstable-2024-04-13";
```

```suggestion
  version = "0-unstable-2024-04-13";
```
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming

---

I had not seen that the naming scheme had changed. Thank you!